README.md

```diff
@@ -16,258 +16,467 @@
 
 Translated in [🇰🇷 Korean](README_KR.md)/
 
-Ludwig is a data-centric deep learning framework that allows users to train and test deep learning models by specifying a declarative configuration that matches the schema of the data.
-It is built on top of PyTorch.
+# What is Ludwig?
 
-To train a model all you need to provide is a file containing your data, a list of columns to use as inputs, and a list of columns to use as outputs, Ludwig will do the rest.
-Simple commands can be used to train models both locally and in a distributed way, and to use them to predict new data.
+Ludwig is a data-driven declarative ML framework hosted by the LF AI & Data Foundation that makes it easy to define
+deep learning pipelines for many types of tasks with a simple and flexible data-driven configuration system.
 
-A programmatic API is also available to use Ludwig from Python.
-A suite of visualization tools allows you to analyze models' training and test performance and to compare them.
+![img](https://raw.githubusercontent.com/ludwig-ai/ludwig-docs/ludwig05/docs/images/ludwig_legos.gif)
 
-Ludwig is built with extensibility principles in mind and is based on datatype abstractions, making it easy to add support for new datatypes as well as new model architectures.
+Ludwig supports many different combinations of input and output types that solve many different machine learning tasks.
```

Comment:
This is proably a nit...replace the second occurrence of "many different" with "diverse"...something like this.

> Ludwig supports many different combinations of input and output types that solve diverse machine learning tasks.